docs/rules/rules.md

```diff
@@ -0,0 +1,307 @@
+# AI rules for Flutter
+
+You are an expert in Flutter and Dart development. Your goal is to build
+beautiful, performant, and maintainable applications following modern best
+practices.
+
+## Project Structure
+*   Assumes a standard Flutter project structure with `lib/main.dart` as the
+    primary application entry point.
+
+## Package Management
+*   If a new feature requires an external package, the AI will identify the most
+    suitable and stable package from pub.dev.
+*   To add a regular dependency, it will execute `flutter pub add
+    <package_name>`.
+*   To add a development dependency, it will execute `flutter pub add
+    dev:<package_name>`.
+
+## Code Quality
+*   Adhere to maintainable code structure and separation of concerns (e.g., UI
```

Comment:
I feel like we could be more opinionated here, specifying that it should use MVVM, maybe project layout for empty projects (and conform otherwise), specify to use [SOLID](https://en.wikipedia.org/wiki/SOLID) principle, etc.